### PR TITLE
TurboHook V.1.1.0 Update

### PR DESCRIPTION
**Pull request by CubesterYT**
Hello again! I'm putting up the PR again because I did some fatal merges to the master branch of my fork, causing me to have to delete my entire fork. So using my backups, I remade the PR. So let me reiterate the last PR:

The moment everyone's been waiting for... TurboHook V.1.1.0!!!

This update includes full embed support, JSON support, bugfixes, typo fixes, some semi colons, extra space removals, and so much more.

Here is the current roster:
image

### CODE REVIEW

File: extensions/CubesterYT/TurboHook.js [hunk 1]
```diff
@@ -85,9 +138,95 @@
...
+        const color = JSON.stringify(hexToDecimal(DATA));
+        return JSON.stringify({ color: color });
+      } else {
+        return "{}";
+      }
+    }
+    embedConnector ({STRING1, STRING2}) {
+      STRING1 = Scratch.Cast.toString(STRING1);
+      STRING2 = Scratch.Cast.toString(STRING2);
+      if (STRING1.includes("author") && STRING2.includes("author")) {
```
Comment (GarboMuffin) [inline extensions/CubesterYT/TurboHook.js:199]: making the same mistakes as you did in the original version

you can make this work without duplicating all sorts of code

**Comment by GarboMuffin** [comment]
the documentation should contain some actual examples at this point

going to be honest, the entire system of reporters and connectors you have seems to just add a whole lot of complexity that a couple "set text to [ ]" blocks would've avoided...

File: extensions/CubesterYT/TurboHook.js (hunk 1 above)
Comment (GarboMuffin) [reply extensions/CubesterYT/TurboHook.js:199]: believe what you want is something like "js deep merge objects", but also be careful of "prototype pollution" bugs

**Comment by CubesterYT** [comment]
These days, I've been thinking about the future of TurboHook. Since TurboHook was meant for webhooks, it should just be webhooks. Nothing specific. I've been thinking of overhauling the entire system for TurboHook. After I've had people legitimately use it for webhooks other than those of Discord, I've been thinking of changing the entire setup of TurboHook to have 1 - 3 simple blocks meant for webhooks. I've been thinking of it ever since I added JSON support. So I might take down this PR and put up a V.2.0.0 PR for TurboHook with an entire overhaul of it. Just a quick question, all I have to do to remove previous blocks is just add "hideFromPalatte = true"?

**Comment by CubesterYT** [comment]
Sorry, didn't mean to request a review here.

**Comment by BlueDome77** [comment]
> all I have to do to remove previous blocks is just add "hideFromPalatte = true"?

Correct

**Comment by CubesterYT** [comment]
> > all I have to do to remove previous blocks is just add "hideFromPalatte = true"?
> 
> Correct

Lol you didn't have to tell me, I found out some time later.

**Pull request closed without merging**